### ISSUE TITLE
[Azure Toolkit][Blocking Issue][A11yMAS][Accessibility]High Contrast][MAS4.3.1][IntelliJ][Wipro]:  High Contrast Black and White OS settings are not getting applied for IntelliJ IDEA Application

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1356]
**Repro Steps:**
**Pre-Requisite**: High Contrast  settings should be enabled

1.  Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA.
3. Navigate to Tools menu, point to "Azure", and then select "Azure Sign In" then observe IntelliJ idea in both high contrast black and white modes

**Expected Result:**
Application should be supported High Contrast black and White OS settings for IntelliJ IDEA. 
**Actual Result:**
High Contrast black and White OS settings are not getting applied for  IntelliJ IDEA.

**Platform:**
IntelliJ IDEA 2017.3.2(Community Edition)
Build #IC-173.4127.27
JRE:1.8.0_152-release-1024-b8 amd64
OS version : 1709
OS build : 16299.192

![image](https://user-images.githubusercontent.com/22956130/35498237-771852e6-04f3-11e8-81bc-9557a2fc82c5.png)

**Comment by Eskibear** [comment]
Similar issue tracked here:
https://youtrack.jetbrains.com/issue/IDEA-41880

**Comment by soumyatippapuram** [comment]
This issue is still reproducible in current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749826
